### ISSUE TITLE
[Suggestion] Config option to disable Nuclearcraft "crushed quartz"

### ISSUE THREAD

**Issue by brdagr** [turbodiesel4598/NuclearCraft#277]
Hello, I am playing on FTB Revelations 1.8.0 version 2.9e--1.12.2 and I am trying to pulverize Nether Quartz into Nether Quartz Dust (from AE2) so I can smelt it into silicon. Nuclearcraft overrides these recipes and does not have a config option to disable either the pulverizer or furnace recipes.  Additionally, either Nuclearcraft or something else is disabling the AE2 "crushed nether quartz" item from being in the game.

It would be nice to have a config option to either not get "crushed quartz" from pulverizing Nether Quartz, or to disable the furnace recipe of "crushed quartz" turning back into nether quartz, and just let AE2s oredict tell the furnace to turn the dust into silicon.

Thank you for your time.

**Comment by brdagr** [comment]
Hi, sorry that I made the issue in the first place. I thought because I wanted to smelt nether quartz dust into silicon that I needed to disable the nuclearcraft dust that was preventing it, but there is a simple workaround by using a crafttweaker script to change around some recipes. Otherwise your dust plays fine with other mods because its in the oreDict, so thank you for your time and again sorry for creating this issue in the first place! :)
